docs/localdatetime-draft.md

```diff
@@ -0,0 +1,96 @@
+  If that is so common, then we may as well have a type for it.
+- Even worse, there is a temptation to pass around a serialized string instead of such a record, which is wrong because the time zone offset may have changed when you read in the string again with `Temporal.Absolute.from()`.
+
+**API**: The API looks something like this:
+
+```typescript
+class Temporal.LocalDateTime {
+  // creation
+  constructor(absolute: Temporal.Absolute, timeZone: Temporal.TimeZone, calendar: Temporal.Calendar);
+  static from(item: string | object, options?: object) : Temporal.LocalDateTime;
+
+  // immutable 'mutation'
+  with(localDateTimeLike: object, options?: object) : Temporal.LocalDateTime;
+  withCalendar(calendar: Temporal.Calendar) : Temporal.LocalDateTime;
+
+  // getters / field access
+  get era(): string | undefined;
+  get year(): number;
+  get month(): number;
+  get day(): number;
+  get hour(): number;
+  get minute(): number;
+  get second(): number;
+  get millisecond(): number;
+  get microsecond(): number;
+  get nanosecond(): number;
+  get timeZone(): Temporal.TimeZone;
+  get timeZoneOffsetNanoseconds(): number;
+  get timeZoneOffsetString(): string;
+  get calendar(): Temporal.Calendar;
+  get dayOfWeek(): number;
+  get dayOfYear(): number;
+  get weekOfYear(): number;
+  get hoursInDay(): number;
+  get daysInMonth(): number;
+  get daysInYear(): number;
+  get isLeapYear(): boolean;
+  getFields(): object;
+  getISOCalendarFields(): object;
+
+  // type conversion
+  toAbsolute(): Temporal.Absolute;
+  toDateTime(): Temporal.DateTime;
+  toDate(): Temporal.Date;
+  toYearMonth(): Temporal.YearMonth;
+  toMonthDay(): Temporal.MonthDay;
+  toTime(): Temporal.Time;
+
+  // comparison
+  static compare(one: Temporal.LocalDateTime, two: Temporal.LocalDateTime): number;
+  equals(other: Temporal.LocalDateTime): boolean;
+
+  // arithmetic
+  plus(duration: Temporal.Duration | object, options?: object) : Temporal.LocalDateTime;
+  minus(duration: Temporal.Duration | object, options?: object) : Temporal.LocalDateTime;
+  difference(other: Temporal.LocalDateTime, options?: object) : Temporal.Duration;
+
+  // serialization / presentation
+  toString() : string;
+  toJSON() : string;
+  toLocaleString(locales?: string | Array<string>, options?: object): string;
+}
+```
+
+(This is not a fully functional description, it's made concise for readability.
+For example, the details of the options bags are omitted.)
+
+It basically has all the API of DateTime, but with a time zone as well.
+But since it represents an unambiguous moment in time (like Absolute, and unlike DateTime), the internal model is more like Absolute with a time zone and calendar.
+
+To create one, you use `Temporal.LocalDateTime.from`.
+- From a string: `from(string)`
+- From an Absolute: `from({ absolute, timeZone })`
```

Comment:
AFAIK this syntax has not been approved. Per our 2020-07-31 meeting I removed the `absolute` field from LocalDateTime in favor of `timeZoneOffsetNanoseconds`. 

Instead, the syntax that seems to have consensus for these use cases is `toLocalDateTime()` on Absolute, DateTime, Date, and Time. #700's poc.d.ts has the signatures of these methods.

Reply:
You're right; I started writing this a while ago and didn't update it. Should be fixed now.